pkcs11test.cc

```diff
@@ -83,6 +83,7 @@ void usage() {
   cerr << "  -m name : name of PKCS#11 library" << endl;
   cerr << "  -l path : path to PKCS#11 library" << endl;
   cerr << "  -s id   : slot ID to perform tests against" << endl;
+  cerr << "  -S n    : slot index (0 for first slot)" << endl;
```

Comment:
It would be nice to expand the help text just a smidgeon, maybe "slot index to use (0 for first); useful if slot IDs are not stable" or somesuch?